config-sample.yaml

```diff
@@ -107,6 +107,17 @@ rtc:
   # # are disabled and clients don't ACK opened peer connections, only reliable, ordered delivery
   # # will be available.
   # strict_acks: true
+  # #
+  # # Settings for improving throughput in high stress forwarding load case.
+  # #
+  # # when set, Livekit will use UDP mux to listen on udp ports (up to runtime.NumCPU()) to
+  # # balance traffic between those ports, also superior to udp_port
+  # udp_ports: 7882-7892
+  # # enable batch write to merge network writting system calls to reduce cpu usage. Outgoing packet
+  # # will be queued until length of queue equal to `batch_size` or time elapsed since last write exceeds `max_flush_interval`.
```

Comment:
`equal to` -> `>=` (technically I guess it can get bigger if the check is only every 1/2 interval, right?)

Reply:
It flushes the queue when it is full so there is no greater. I think the ticker interval doesn't need to be divided by 2, will change it.